projects/scripts/adi_board.tcl

```diff
@@ -517,27 +520,24 @@ proc ad_mem_hpx_interconnect {p_sel p_clk p_name} {
     set_property CONFIG.NUM_SI 1 $m_interconnect_cell
     ad_connect $p_rst $m_interconnect_cell/ARESETN
     ad_connect $p_clk $m_interconnect_cell/ACLK
-    ad_connect $p_rst $m_interconnect_cell/M00_ARESETN
-    ad_connect $p_clk $m_interconnect_cell/M00_ACLK
     ad_connect $m_interconnect_cell/M00_AXI $p_name_int
     if {$p_intf_clock ne ""} {
       ad_connect $p_clk $p_intf_clock
     }
   } else {
     set_property CONFIG.NUM_SI $m_interconnect_index $m_interconnect_cell
-    ad_connect $p_rst $m_interconnect_cell/${i_str}_ARESETN
-    ad_connect $p_clk $m_interconnect_cell/${i_str}_ACLK
+    if {[lsearch [get_bd_nets -of_object [get_bd_pins $m_interconnect_cell/ACLK*]] "/$p_clk"] == -1 } {
+        set sys_mem_clk_index [expr $sys_mem_clk_index + 1]
+        set_property CONFIG.NUM_CLKS [expr $sys_mem_clk_index +1] $m_interconnect_cell
+        ad_connect $p_clk $m_interconnect_cell/ACLK[expr $sys_mem_clk_index]
```

Comment:
I think we don't need the expr in this case, just use '$m_interconnect_cell/ACLK$sys_mem_clk_index'.